src/templates/partials/blog-card.njk

```diff
@@ -2,14 +2,11 @@
   {% for blog in blog.blogCards %}
     {% set itemMod = '' %}
     {% set imageMod = '' %}
-    {% if loop.first %}
-      {% set itemMod = 'first-span' %}
-    {% endif %}
-    {% if loop.last %}
-      {% set itemMod = 'last-span' %}
+    {% if loop.index in[1, 5] %}
```

Comment:
Last suggestion here is to use a trigger, it should be in the data that if we have a large card, then we give the modifier `large`, and not by index. It's the same with image modifiers like `pos20` or `pos40`